diff -Naur RBDOOM-3-BFG-435637606d65efe1098683d133f348eb06f8b852.old/neo/CMakeLists.txt RBDOOM-3-BFG-435637606d65efe1098683d133f348eb06f8b852/neo/CMakeLists.txt
--- RBDOOM-3-BFG-435637606d65efe1098683d133f348eb06f8b852.old/neo/CMakeLists.txt	2019-01-06 17:21:21.107807212 +0100
+++ RBDOOM-3-BFG-435637606d65efe1098683d133f348eb06f8b852/neo/CMakeLists.txt	2019-01-06 17:22:11.982750434 +0100
@@ -1374,6 +1374,7 @@
 		dbghelp
 		#eaxguid
 		iphlpapi
+        minizip
 		winmm
 		wsock32.lib
 		${OpenAL_LIBRARIES}
@@ -1542,6 +1543,7 @@
 		target_link_libraries(RBDoom3BFG
 			idlib
 			${OPENGL_LIBRARIES}
+            minizip
 			pthread
 			${RT_LIBRARY}
 			${SDLx_LIBRARY}
diff -Naur RBDOOM-3-BFG-435637606d65efe1098683d133f348eb06f8b852.old/neo/framework/Unzip.h RBDOOM-3-BFG-435637606d65efe1098683d133f348eb06f8b852/neo/framework/Unzip.h
--- RBDOOM-3-BFG-435637606d65efe1098683d133f348eb06f8b852.old/neo/framework/Unzip.h	2019-01-06 17:21:21.119807434 +0100
+++ RBDOOM-3-BFG-435637606d65efe1098683d133f348eb06f8b852/neo/framework/Unzip.h	2019-01-06 17:22:11.982750434 +0100
@@ -37,14 +37,6 @@
 #ifndef __UNZIP_H__
 #define __UNZIP_H__
 
-#include "../libs/zlib/minizip/unzip.h"
-
-/*
- * DG: all unzip code has been moved to minizip/
- *     when being updated to minizip 1.1 (instead of the previously used minizip 0.15beta from 1998)
- *     This update brings zip64 support (untested) and much cleaner code.
- *     Furthermore, updates to future minizip should be much easier now, as my changes to the
- *     original source files from minizip are quite small.
- */
+#include <minizip/unzip.h>
 
 #endif /* __UNZIP_H__ */
diff -Naur RBDOOM-3-BFG-435637606d65efe1098683d133f348eb06f8b852.old/neo/framework/Zip.h RBDOOM-3-BFG-435637606d65efe1098683d133f348eb06f8b852/neo/framework/Zip.h
--- RBDOOM-3-BFG-435637606d65efe1098683d133f348eb06f8b852.old/neo/framework/Zip.h	2019-01-06 17:21:21.119807434 +0100
+++ RBDOOM-3-BFG-435637606d65efe1098683d133f348eb06f8b852/neo/framework/Zip.h	2019-01-06 17:22:11.983750452 +0100
@@ -30,8 +30,7 @@
 
 #include <zlib.h>
 
-// DG: all the zip access stuff from minizip is now in minizip/zip.h
-#include "libs/zlib/minizip/zip.h"
+#include <minizip/zip.h>
 
 
 /*
