diff -Naur RBDOOM-3-BFG-a5a6bf0e33d985a2af47b8db2a000200617179c5.old/neo/CMakeLists.txt RBDOOM-3-BFG-a5a6bf0e33d985a2af47b8db2a000200617179c5/neo/CMakeLists.txt
--- RBDOOM-3-BFG-a5a6bf0e33d985a2af47b8db2a000200617179c5.old/neo/CMakeLists.txt	2023-03-15 10:48:25.862916983 +0100
+++ RBDOOM-3-BFG-a5a6bf0e33d985a2af47b8db2a000200617179c5/neo/CMakeLists.txt	2023-03-15 10:49:17.722803822 +0100
@@ -1612,6 +1612,7 @@
 		dbghelp
 		#eaxguid
 		iphlpapi
+        minizip
 		winmm
 		wsock32.lib
 		${Vulkan_LIBRARIES}
@@ -1817,6 +1818,7 @@
 			${Vulkan_LIBRARY}
 			#${OPENGL_LIBRARIES}
 			pthread
+            minizip
 			${RT_LIBRARY}
 			${SDLx_LIBRARY}
 			${OPENAL_LIBRARY}
diff -Naur RBDOOM-3-BFG-a5a6bf0e33d985a2af47b8db2a000200617179c5.old/neo/framework/Unzip.h RBDOOM-3-BFG-a5a6bf0e33d985a2af47b8db2a000200617179c5/neo/framework/Unzip.h
--- RBDOOM-3-BFG-a5a6bf0e33d985a2af47b8db2a000200617179c5.old/neo/framework/Unzip.h	2023-03-15 10:48:25.890917461 +0100
+++ RBDOOM-3-BFG-a5a6bf0e33d985a2af47b8db2a000200617179c5/neo/framework/Unzip.h	2023-03-15 10:48:50.720342060 +0100
@@ -37,14 +37,6 @@
 #ifndef __UNZIP_H__
 #define __UNZIP_H__
 
-#include "../libs/zlib/minizip/unzip.h"
-
-/*
- * DG: all unzip code has been moved to minizip/
- *     when being updated to minizip 1.1 (instead of the previously used minizip 0.15beta from 1998)
- *     This update brings zip64 support (untested) and much cleaner code.
- *     Furthermore, updates to future minizip should be much easier now, as my changes to the
- *     original source files from minizip are quite small.
- */
+#include <minizip/unzip.h>
 
 #endif /* __UNZIP_H__ */
diff -Naur RBDOOM-3-BFG-a5a6bf0e33d985a2af47b8db2a000200617179c5.old/neo/framework/Zip.h RBDOOM-3-BFG-a5a6bf0e33d985a2af47b8db2a000200617179c5/neo/framework/Zip.h
--- RBDOOM-3-BFG-a5a6bf0e33d985a2af47b8db2a000200617179c5.old/neo/framework/Zip.h	2023-03-15 10:48:25.891917478 +0100
+++ RBDOOM-3-BFG-a5a6bf0e33d985a2af47b8db2a000200617179c5/neo/framework/Zip.h	2023-03-15 10:48:51.581356784 +0100
@@ -30,9 +30,8 @@
 
 #include <zlib.h>
 
-// DG: all the zip access stuff from minizip is now in minizip/zip.h
 #ifndef TYPEINFOPROJECT
-	#include "libs/zlib/minizip/zip.h"
+	#include <minizip/zip.h>
 #endif
 
 
