diff -Naur RBDOOM-3-BFG-1.6.0.old/neo/CMakeLists.txt RBDOOM-3-BFG-1.6.0/neo/CMakeLists.txt
--- RBDOOM-3-BFG-1.6.0.old/neo/CMakeLists.txt	2026-01-09 18:52:16.714619965 +0100
+++ RBDOOM-3-BFG-1.6.0/neo/CMakeLists.txt	2026-01-09 18:53:35.533493741 +0100
@@ -453,7 +453,7 @@
 
 # RB: there haven't been requests for a separate map compiler executable by macOS users
 # and it is ok to not support it until someone also supports TrenchBroomBFG on macOS
-if(NOT APPLE AND NOT FLATPAK)
+if(NOT APPLE AND NOT FLATPAK AND NOT LINUX)
 	add_subdirectory(tools/compilers)
 endif()
 
