diff -Naur RBDOOM-3-BFG-18755609de07ab87dea78b8cd2650435909651a1.old/neo/idlib/Thread.cpp RBDOOM-3-BFG-18755609de07ab87dea78b8cd2650435909651a1/neo/idlib/Thread.cpp
--- RBDOOM-3-BFG-18755609de07ab87dea78b8cd2650435909651a1.old/neo/idlib/Thread.cpp	2024-04-06 18:40:04.335453832 +0200
+++ RBDOOM-3-BFG-18755609de07ab87dea78b8cd2650435909651a1/neo/idlib/Thread.cpp	2024-04-06 18:40:14.425632780 +0200
@@ -255,9 +255,6 @@
 	catch( idException& ex )
 	{
 		idLib::Warning( "Fatal error in thread %s: %s", thread->GetName(), ex.GetError() );
-
-		// We don't handle threads terminating unexpectedly very well, so just terminate the whole process
-		exit( 0 );
 	}
 
 	thread->isRunning = false;
