diff -Naur RBDOOM-3-BFG-80df3182f1db17c282306f4095a75637db86f484.old/neo/CMakeLists.txt RBDOOM-3-BFG-80df3182f1db17c282306f4095a75637db86f484/neo/CMakeLists.txt
--- RBDOOM-3-BFG-80df3182f1db17c282306f4095a75637db86f484.old/neo/CMakeLists.txt	2017-01-09 14:33:24.150178668 +0100
+++ RBDOOM-3-BFG-80df3182f1db17c282306f4095a75637db86f484/neo/CMakeLists.txt	2017-01-09 14:33:56.436813218 +0100
@@ -1342,6 +1342,7 @@
 		dbghelp
 		#eaxguid
 		iphlpapi
+        minizip
 		winmm
 		wsock32.lib
 		${OpenAL_LIBRARIES}
@@ -1510,6 +1511,7 @@
 		target_link_libraries(RBDoom3BFG
 			idlib
 			${OPENGL_LIBRARIES}
+            minizip
 			pthread
 			${RT_LIBRARY}
 			${SDLx_LIBRARY}
diff -Naur RBDOOM-3-BFG-80df3182f1db17c282306f4095a75637db86f484.old/neo/framework/Unzip.h RBDOOM-3-BFG-80df3182f1db17c282306f4095a75637db86f484/neo/framework/Unzip.h
--- RBDOOM-3-BFG-80df3182f1db17c282306f4095a75637db86f484.old/neo/framework/Unzip.h	2017-01-09 14:33:24.162178904 +0100
+++ RBDOOM-3-BFG-80df3182f1db17c282306f4095a75637db86f484/neo/framework/Unzip.h	2017-01-09 14:33:56.436813218 +0100
@@ -37,14 +37,6 @@
 #ifndef __UNZIP_H__
 #define __UNZIP_H__
 
-#include "../libs/zlib/minizip/unzip.h"
-
-/*
- * DG: all unzip code has been moved to minizip/
- *     when being updated to minizip 1.1 (instead of the previously used minizip 0.15beta from 1998)
- *     This update brings zip64 support (untested) and much cleaner code.
- *     Furthermore, updates to future minizip should be much easier now, as my changes to the
- *     original source files from minizip are quite small.
- */
+#include <minizip/unzip.h>
 
 #endif /* __UNZIP_H__ */
diff -Naur RBDOOM-3-BFG-80df3182f1db17c282306f4095a75637db86f484.old/neo/framework/Zip.h RBDOOM-3-BFG-80df3182f1db17c282306f4095a75637db86f484/neo/framework/Zip.h
--- RBDOOM-3-BFG-80df3182f1db17c282306f4095a75637db86f484.old/neo/framework/Zip.h	2017-01-09 14:33:24.162178904 +0100
+++ RBDOOM-3-BFG-80df3182f1db17c282306f4095a75637db86f484/neo/framework/Zip.h	2017-01-09 14:33:56.437813238 +0100
@@ -30,8 +30,7 @@
 
 #include <zlib.h>
 
-// DG: all the zip access stuff from minizip is now in minizip/zip.h
-#include "libs/zlib/minizip/zip.h"
+#include <minizip/zip.h>
 
 
 /*
