diff -Naur RBDoom3BFG.old/neo/idlib/Thread.cpp RBDoom3BFG/neo/idlib/Thread.cpp
--- RBDoom3BFG.old/neo/idlib/Thread.cpp	2013-11-15 16:50:34.532416813 +0100
+++ RBDoom3BFG/neo/idlib/Thread.cpp	2013-11-15 17:32:25.425805095 +0100
@@ -249,9 +249,6 @@
 	catch( idException& ex )
 	{
 		idLib::Warning( "Fatal error in thread %s: %s", thread->GetName(), ex.GetError() );
-		
-		// We don't handle threads terminating unexpectedly very well, so just terminate the whole process
-		exit( 0 );
 	}
 	
 	thread->isRunning = false;
