diff -Naur RBDOOM-3-BFG-1.4.0.old/neo/CMakeLists.txt RBDOOM-3-BFG-1.4.0/neo/CMakeLists.txt
--- RBDOOM-3-BFG-1.4.0.old/neo/CMakeLists.txt	2022-04-08 09:31:09.199187264 +0200
+++ RBDOOM-3-BFG-1.4.0/neo/CMakeLists.txt	2022-04-08 09:31:27.388500044 +0200
@@ -1657,6 +1657,7 @@
 		dbghelp
 		#eaxguid
 		iphlpapi
+        minizip
 		winmm
 		wsock32.lib
 		${Vulkan_LIBRARIES}
@@ -1858,6 +1859,7 @@
 			idlib
 			${Vulkan_LIBRARIES}
 			${OPENGL_LIBRARIES}
+            minizip
 			pthread
 			${RT_LIBRARY}
 			${SDLx_LIBRARY}
diff -Naur RBDOOM-3-BFG-1.4.0.old/neo/framework/Unzip.h RBDOOM-3-BFG-1.4.0/neo/framework/Unzip.h
--- RBDOOM-3-BFG-1.4.0.old/neo/framework/Unzip.h	2022-04-08 09:31:09.212187487 +0200
+++ RBDOOM-3-BFG-1.4.0/neo/framework/Unzip.h	2022-04-08 09:31:27.388500044 +0200
@@ -37,14 +37,6 @@
 #ifndef __UNZIP_H__
 #define __UNZIP_H__
 
-#include "../libs/zlib/minizip/unzip.h"
-
-/*
- * DG: all unzip code has been moved to minizip/
- *     when being updated to minizip 1.1 (instead of the previously used minizip 0.15beta from 1998)
- *     This update brings zip64 support (untested) and much cleaner code.
- *     Furthermore, updates to future minizip should be much easier now, as my changes to the
- *     original source files from minizip are quite small.
- */
+#include <minizip/unzip.h>
 
 #endif /* __UNZIP_H__ */
diff -Naur RBDOOM-3-BFG-1.4.0.old/neo/framework/Zip.h RBDOOM-3-BFG-1.4.0/neo/framework/Zip.h
--- RBDOOM-3-BFG-1.4.0.old/neo/framework/Zip.h	2022-04-08 09:31:09.212187487 +0200
+++ RBDOOM-3-BFG-1.4.0/neo/framework/Zip.h	2022-04-08 09:31:27.389500061 +0200
@@ -30,9 +30,8 @@
 
 #include <zlib.h>
 
-// DG: all the zip access stuff from minizip is now in minizip/zip.h
 #ifndef TYPEINFOPROJECT
-	#include "libs/zlib/minizip/zip.h"
+	#include <minizip/zip.h>
 #endif
 
 
