diff -Naur RBDOOM-3-BFG-880667da5431a9be602030a0d004815543807ac5.old/neo/CMakeLists.txt RBDOOM-3-BFG-880667da5431a9be602030a0d004815543807ac5/neo/CMakeLists.txt
--- RBDOOM-3-BFG-880667da5431a9be602030a0d004815543807ac5.old/neo/CMakeLists.txt	2019-11-03 15:18:36.109641001 +0100
+++ RBDOOM-3-BFG-880667da5431a9be602030a0d004815543807ac5/neo/CMakeLists.txt	2019-11-03 15:18:45.667826410 +0100
@@ -1484,6 +1484,7 @@
 		dbghelp
 		#eaxguid
 		iphlpapi
+        minizip
 		winmm
 		wsock32.lib
 		${Vulkan_LIBRARIES}
@@ -1673,6 +1674,7 @@
 		target_link_libraries(RBDoom3BFG
 			idlib
 			${OPENGL_LIBRARIES}
+            minizip
 			pthread
 			${RT_LIBRARY}
 			${SDLx_LIBRARY}
diff -Naur RBDOOM-3-BFG-880667da5431a9be602030a0d004815543807ac5.old/neo/framework/Unzip.h RBDOOM-3-BFG-880667da5431a9be602030a0d004815543807ac5/neo/framework/Unzip.h
--- RBDOOM-3-BFG-880667da5431a9be602030a0d004815543807ac5.old/neo/framework/Unzip.h	2019-11-03 15:18:36.123641273 +0100
+++ RBDOOM-3-BFG-880667da5431a9be602030a0d004815543807ac5/neo/framework/Unzip.h	2019-11-03 15:18:45.667826410 +0100
@@ -37,14 +37,6 @@
 #ifndef __UNZIP_H__
 #define __UNZIP_H__
 
-#include "../libs/zlib/minizip/unzip.h"
-
-/*
- * DG: all unzip code has been moved to minizip/
- *     when being updated to minizip 1.1 (instead of the previously used minizip 0.15beta from 1998)
- *     This update brings zip64 support (untested) and much cleaner code.
- *     Furthermore, updates to future minizip should be much easier now, as my changes to the
- *     original source files from minizip are quite small.
- */
+#include <minizip/unzip.h>
 
 #endif /* __UNZIP_H__ */
diff -Naur RBDOOM-3-BFG-880667da5431a9be602030a0d004815543807ac5.old/neo/framework/Zip.h RBDOOM-3-BFG-880667da5431a9be602030a0d004815543807ac5/neo/framework/Zip.h
--- RBDOOM-3-BFG-880667da5431a9be602030a0d004815543807ac5.old/neo/framework/Zip.h	2019-11-03 15:18:36.123641273 +0100
+++ RBDOOM-3-BFG-880667da5431a9be602030a0d004815543807ac5/neo/framework/Zip.h	2019-11-03 15:18:45.723827497 +0100
@@ -30,8 +30,7 @@
 
 #include <zlib.h>
 
-// DG: all the zip access stuff from minizip is now in minizip/zip.h
-#include "libs/zlib/minizip/zip.h"
+#include <minizip/zip.h>
 
 
 /*
