diff -Naur RBDOOM-3-BFG-435637606d65efe1098683d133f348eb06f8b852.old/neo/idlib/precompiled.h RBDOOM-3-BFG-435637606d65efe1098683d133f348eb06f8b852/neo/idlib/precompiled.h
--- RBDOOM-3-BFG-435637606d65efe1098683d133f348eb06f8b852.old/neo/idlib/precompiled.h	2019-01-06 17:21:21.124807527 +0100
+++ RBDOOM-3-BFG-435637606d65efe1098683d133f348eb06f8b852/neo/idlib/precompiled.h	2019-01-06 17:24:37.306444763 +0100
@@ -105,7 +105,7 @@
 #include "../ui/UserInterface.h"
 
 // RB: required for SWF extensions
-#include "rapidjson/document.h"
+#include <rapidjson/document.h>
 
 #include "../swf/SWF.h"
 
diff -Naur RBDOOM-3-BFG-435637606d65efe1098683d133f348eb06f8b852.old/neo/swf/SWF_Load.cpp RBDOOM-3-BFG-435637606d65efe1098683d133f348eb06f8b852/neo/swf/SWF_Load.cpp
--- RBDOOM-3-BFG-435637606d65efe1098683d133f348eb06f8b852.old/neo/swf/SWF_Load.cpp	2019-01-06 17:21:21.228809455 +0100
+++ RBDOOM-3-BFG-435637606d65efe1098683d133f348eb06f8b852/neo/swf/SWF_Load.cpp	2019-01-06 17:24:04.294832723 +0100
@@ -30,7 +30,7 @@
 #include "precompiled.h"
 #include "../renderer/Font.h"
 #include "../renderer/Image.h"
-//#include "../../libs/rapidjson/include/rapidjson/document.h"
+//#include <include/rapidjson/document.h>
 
 using namespace rapidjson;
 
